sdk/python/feast/infra/registry/sql.py

```diff
@@ -181,7 +181,7 @@ class FeastMetadataKeys(Enum):
 
 class SqlRegistry(BaseRegistry):
     def __init__(
-        self, registry_config: Optional[RegistryConfig], repo_path: Optional[Path]
+        self, registry_config: Optional[RegistryConfig], repo_path: Optional[Path], is_feast_apply: False,
```

Comment:
@IsraelAgundis We can probably make this optional?

Reply:
Yes, but why?

From my understanding, `Optional[bool]` is usually reserved when you care about whether that variable is:
1. True
2. False
3. None

In this case, we only care if the variable is:
1. True
2. False

I'll let you have the final say though, so lmk if I should add it.